downloader/src/main/kotlin/vcs/Mercurial.kt

```diff
@@ -65,7 +65,7 @@ class Mercurial : VersionControlSystem() {
     override fun getWorkingTree(vcsDirectory: File) = MercurialWorkingTree(vcsDirectory, type)
 
     override fun isApplicableUrlInternal(vcsUrl: String) =
-        ProcessCapture("hg", "identify", vcsUrl).isSuccess
+        MercurialCommand.isInPath() && ProcessCapture("hg", "identify", vcsUrl).isSuccess
```

Comment:
commit: IIUC all production code callers only call visit this code path only if `VersionControlSystem.isAvailable()` returns `true`. Would you mind providing the callstack which leads to the mentioned exception to help me understanding it better?